Electronics/PCB/AstroPlant_ExtensionBoard/astroplant-extension-board-mk0-7/Library/SamacSys_Parts.pretty/LQG15WH0N7B02D.kicad_mod

```diff
@@ -0,0 +1,30 @@
+(module "LQG15WH0N7B02D" (layer F.Cu)
+  (descr "0402 (1005) T=0.5±0.05mm")
+  (tags "Inductor")
+  (attr smd)
+  (fp_text reference L** (at 0.000 -0) (layer F.SilkS)
+    (effects (font (size 1.27 1.27) (thickness 0.254)))
+  )
+  (fp_text user %R (at 0.000 -0) (layer F.Fab)
+    (effects (font (size 1.27 1.27) (thickness 0.254)))
+  )
+  (fp_text value "LQG15WH0N7B02D" (at 0.000 -0) (layer F.SilkS) hide
+    (effects (font (size 1.27 1.27) (thickness 0.254)))
+  )
+  (fp_line (start -1.175 -0.55) (end 1.175 -0.55) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 1.175 -0.55) (end 1.175 0.55) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 1.175 0.55) (end -1.175 0.55) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -1.175 0.55) (end -1.175 -0.55) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -0.5 -0.25) (end 0.5 -0.25) (layer F.Fab) (width 0.1))
+  (fp_line (start 0.5 -0.25) (end 0.5 0.25) (layer F.Fab) (width 0.1))
+  (fp_line (start 0.5 0.25) (end -0.5 0.25) (layer F.Fab) (width 0.1))
+  (fp_line (start -0.5 0.25) (end -0.5 -0.25) (layer F.Fab) (width 0.1))
+  (fp_line (start 0 -0.25) (end 0 0.25) (layer F.SilkS) (width 0.1))
+  (pad 1 smd rect (at -0.425 -0 0) (size 0.350 0.650) (layers F.Cu F.Paste F.Mask))
+  (pad 2 smd rect (at 0.425 -0 0) (size 0.350 0.650) (layers F.Cu F.Paste F.Mask))
+  (model "C:\\Users\\Aurel\\Documents\\Astroplant\\Hardware\\Depository_astroplant_hardware\\astroplant-hardware\\Electronics\\PCB\\AstroPlant_ExtensionBoard\\astroplant-extension-board-mk0-7\\Library\\SamacSys_Parts.3dshapes\\LQG15HS3N6S02D.stp"
```

Comment:
This and other similar lines seem to cause errors when I open it in KiCad. Is it possible to avoid referring to those files or include them without hardcoding the path?

Reply:
We can merge this PR and I will log this as an issue to work on next